services/api/src/utils/mailer.js

```diff
@@ -30,90 +122,70 @@ marked.use({
     link(href, title, text) {
       if (title === '$button$') {
         return `<a href="${href}" class="button" target="_blank"><span class="text">${text}</span></a>`;
+      } else if (href.includes('{{')) {
+        // Links may include template interpolation which
+        // gets escaped so output it unescaped here.
+        return `<a href="${href}">${text}</a>`;
       }
       return false;
     },
   },
 });
 
-const defaultOptions = {
-  appName: config.get('APP_NAME'),
-  appUrl: config.get('APP_URL'),
-  appSupportEmail: config.get('APP_SUPPORT_EMAIL'),
-  appCompanyName: config.get('APP_COMPANY_NAME'),
-  appCompanyAddress: config.get('APP_COMPANY_ADDRESS'),
-  appLogoUrl: config.get('APP_LOGO_URL'),
+// Mustache utils
+
+// Mustache is HTML escaping URLs by default
+// which causes issues so avoid this.
+const escapeHtml = Mustache.escape;
+Mustache.escape = (text) => {
+  if (!text.match(/https?:\/\//)) {
```

Comment:
love it

Reply:
hah. I thought you'd hate it... only after I did this I realized that `{{&something}}` and `{{{something not escaped}}}` were a thing and I forgot about them... that's the "mustache" way to do it, however the fact that I forgot makes me want to keep it... I don't think I trust even myself to always remember this and then on top of that I can easily see non-technical people to be writing email templates (we tend to go nuts with this) and having to teach them that seems silly... feels hacky but I'd like to keep it